eng/common/scripts/logging.ps1

```diff
@@ -0,0 +1,40 @@
+if ($null -eq $isDevOpsRun)
```

Comment:
Doesn't look like you imported this file.

Reply:
Thanks, I have it imported now.